src/Sarif/Sarif.csproj

```diff
@@ -47,6 +47,7 @@
   <ItemGroup>
     <PackageReference Include="System.Collections.Immutable" Version="1.5.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
```

Comment:
>CommandLineParser [](http://example.com/codeflow?start=31&length=17)

why its required here?

Reply:
Actually not sure, but it is the only project reference by Sarif.Driver.csproj. 
![image](https://user-images.githubusercontent.com/119978670/209258004-8546e26c-3468-4560-83ab-0595411c7f8f.png)

Reply:
Sarif.Driver depends on CommandLineParser but Sarif project doesn't have a dependency on it right?

This is new dependency added to Sarif. It is not required in my mind.

Reply:
Verified that the change in Sarif.csproj is not needed. Remove it.